Movies/routes/celebrities.js

```diff
@@ -0,0 +1,103 @@
+var express = require('express');
+var router = express.Router();
+const Celebrity = require ('../models/celebrity');
+
+
+router.get('/', function(req, res, next) {
+
+  Celebrity.find({}, (err , celebrities)=>{
+    if (err){
+      next(err)
+    }else{
+      console.log(celebrities);
+     res.render('celebrities/index', {celebrities: celebrities});
```

Comment:
This is working, but just to let you know, on ES6 if you are passing a property with the same name, you don't need to do the assignation, so you can write something like:
res.render('celebrities/index', {celebrities});